client/post-editor/editor-drawer/index.jsx

```diff
@@ -328,7 +331,8 @@ export default connect(
 		return {
 			canJetpackUseTaxonomies: isJetpackMinimumVersion( state, siteId, '4.1' ),
 			jetpackVersionSupportsSeo: isJetpackMinimumVersion( state, siteId, '4.4-beta1' ),
-			typeObject: getPostType( state, siteId, type )
+			typeObject: getPostType( state, siteId, type ),
+			isSitePrivate: isPrivateSite( state, siteId ),
```

Comment:
Doing cartwheels to avoid [`no-shadow`](http://eslint.org/docs/rules/no-shadow) ? 😛 Seems fine as `isPrivate` too if that would be clearer than the word shuffling.